### DISCUSSION TITLE
Netflow logstash pipleline question

### DISCUSSION THREAD

**Discussion by gobrien1231** [Security-Onion-Solutions/securityonion#6223] [Unsupported Versions] [answered]
Thank you for the great video explaining how to configure netflow with filebeat/logstash. My question is, is this configuration independent of the monitor bond0 interface? i.e. is the netflow collector address the management ip address?

**Comment by tomjones1977** [comment, marked as answer]
I would say that it is.  The NetFlow source needs an IP to send the data to, and your monitoring interface will not have an IP allocated to it.

**Reply by gobrien1231** [reply]
makes sense, thanks!

**Comment by tomjones1977** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/6214
